sdk/core/core-https/documentation/core2.md

```diff
@@ -0,0 +1,69 @@
+# Azure SDK for JS Core 2.0
```

Comment:
Azure Core is the term we have been using more often outside the JS team, so perhaps people may relate to the below better?

```suggestion
# Azure Core 2.0 for JS SDKs
```